crates/red_knot_python_semantic/src/types.rs

```diff
@@ -3006,8 +3053,10 @@ impl<'db> KnownClass {
             "list" => Self::List,
             "slice" => Self::Slice,
             "range" => Self::Range,
+            "property" => Self::Property,
```

Comment:
It happened again. This was missing previously.

Reply:
Time we took action on this comment, I think...

https://github.com/astral-sh/ruff/blob/b9b094869ac6af590c910fde48b4fd3e01601236/crates/red_knot_python_semantic/src/types.rs#L2990-L2992

Reply:
We could derive iter for the enum and add a unit tests over a macro.

Reply:
I think deriving iter requires adding a red-knot dependency on `strum` and `strum_macros`? Not that I object to that if we think it's worth it

Reply:
A "cheap" version of this could be to add a comment before the first and after the last variant in `KnownClass`, both telling you that you absolutely have to make sure to add the variant to `try_from_file_and_name`

Reply:
#16326 for a possible solution!